core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -176,15 +179,17 @@ private[spark] object HighlyCompressedMapStatus {
     // From a compression standpoint, it shouldn't matter whether we track empty or non-empty
```

Comment:
ah I see, I think I hadn't realized this before -- so even after calling `runOptimize`, the memory usage may be different vs. flipped bits?

In any case, I agree, this comment needs to be modified -- along w/ the [comment at line 124](https://github.com/apache/spark/blob/branch-1.5/core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala#L125), but I guess we need to wait for the other details to shake out to figure out exactly how to update them both.